src/screenshot_gui.cpp

```diff
@@ -14,6 +14,8 @@
 #include "widgets/screenshot_widget.h"
 #include "table/strings.h"
 
+extern DrawPixelInfo _screen;
```

Comment:
Where you searching the `gfx_func.h` header?

Reply:
No, I needed `_screen.width` which I believe is defined in `gfx.cpp`, however if I #include that lots of other things break. Is there a better way to do this?

Reply:
Line 64 of gfx_func.h is `extern DrawPixelInfo _screen;`

Reply:
Would it be better to `#include gfx_func.h`?

Reply:
IMHO yes, especially in a cpp file. There might be a reason to make an exception inside a header file if it would pull in too much other stuff, but that is not a concern for cpp files.

Reply:
Done, thanks.